packages/carequality-sdk/src/client/carequality.ts

```diff
@@ -0,0 +1,123 @@
+import BadRequestError from "@metriport/core/util/error/bad-request";
+import axios, { AxiosInstance, AxiosResponse, AxiosStatic } from "axios";
+import { Bundle as Stu3Bundle, bundleSchema } from "../models/bundle";
+import { OrganizationList } from "../models/organization";
+
+const DEFAULT_AXIOS_TIMEOUT_SECONDS = 120;
+const MAX_COUNT = 1000;
+const JSON_FORMAT = "_format=json";
+
+export enum APIMode {
+  dev = "dev",
+  staging = "stage",
+}
+
+export type Options = {
+  axios?: AxiosStatic; // Set axios if it fails to load
+  timeout?: number;
+};
+
+/**
+ * Currently operates on FHIR Stu3 format.
+ */
+export class Carequality {
+  static devUrl = "https://dev-dir-ceq.sequoiadns.org/fhir-stu3/1.0.1";
+  static stagingUrl = "https://stage-dir-ceq.sequoiaproject.org/fhir-stu3/1.0.1";
```

Comment:
We should probably add `private` and `readonly` to these since the client choses the environment w/ the `APIMode`

Reply:
Should we also leave a placeholder for the prod url? even though we don't have it yet

Reply:
I don't really see the point in adding it now, if it cannot be used yet